src/intrinsic/llvm.rs

```diff
@@ -3,134 +3,16 @@ use gccjit::Function;
-            "llvm.x86.avx2.pmul.hr.sw" => "__builtin_ia32_pmulhrsw256",
-            "llvm.x86.avx2.packsswb" => "__builtin_ia32_packsswb256",
-            "llvm.x86.avx2.packssdw" => "__builtin_ia32_packssdw256",
-            "llvm.x86.avx2.packuswb" => "__builtin_ia32_packuswb256",
-            "llvm.x86.avx2.packusdw" => "__builtin_ia32_packusdw256",
-            "llvm.x86.avx2.permd" => "__builtin_ia32_permvarsi256",
-            "llvm.x86.avx2.permps" => "__builtin_ia32_permvarsf256",
-            "llvm.x86.avx2.psad.bw" => "__builtin_ia32_psadbw256",
-            "llvm.x86.avx2.psign.w" => "__builtin_ia32_psignw256",
-            "llvm.x86.avx2.psign.d" => "__builtin_ia32_psignd256",
-            "llvm.x86.avx2.psign.b" => "__builtin_ia32_psignb256",
-            "llvm.x86.avx2.psll.w" => "__builtin_ia32_psllw256",
-            "llvm.x86.avx2.psll.d" => "__builtin_ia32_pslld256",
-            "llvm.x86.avx2.psll.q" => "__builtin_ia32_psllq256",
-            "llvm.x86.avx2.pslli.w" => "__builtin_ia32_psllwi256",
-            "llvm.x86.avx2.pslli.q" => "__builtin_ia32_psllqi256",
-            "llvm.x86.avx2.psllv.d" => "__builtin_ia32_psllv4si",
-            "llvm.x86.avx2.psllv.d.256" => "__builtin_ia32_psllv8si",
-            "llvm.x86.avx2.psllv.q" => "__builtin_ia32_psllv2di",
-            "llvm.x86.avx2.psllv.q.256" => "__builtin_ia32_psllv4di",
-            "llvm.x86.avx2.psra.w" => "__builtin_ia32_psraw256",
-            "llvm.x86.avx2.psra.d" => "__builtin_ia32_psrad256",
-            "llvm.x86.avx2.psrai.w" => "__builtin_ia32_psrawi256",
-            "llvm.x86.avx2.psrai.d" => "__builtin_ia32_psradi256",
-            "llvm.x86.avx2.psrav.d" => "__builtin_ia32_psrav4si",
-            "llvm.x86.avx2.psrav.d.256" => "__builtin_ia32_psrav8si",
-            "llvm.x86.avx2.psrl.w" => "__builtin_ia32_psrlw256",
-            "llvm.x86.avx2.psrl.d" => "__builtin_ia32_psrld256",
-            "llvm.x86.avx2.psrl.q" => "__builtin_ia32_psrlq256",
-            "llvm.x86.avx2.psrlv.d" => "__builtin_ia32_psrlv4si",
-            "llvm.x86.avx2.psrlv.d.256" => "__builtin_ia32_psrlv8si",
-            "llvm.x86.avx2.psrlv.q" => "__builtin_ia32_psrlv2di",
-            "llvm.x86.avx2.psrlv.q.256" => "__builtin_ia32_psrlv4di",
-            "llvm.x86.sse.sqrt.ss" => "__builtin_ia32_sqrtss",
-            "llvm.x86.pclmulqdq" => "__builtin_ia32_pclmulqdq128",
-            "llvm.x86.sha1msg1" => "__builtin_ia32_sha1msg1",
-            "llvm.x86.sha1msg2" => "__builtin_ia32_sha1msg2",
-            "llvm.x86.sha1nexte" => "__builtin_ia32_sha1nexte",
-            "llvm.x86.sha1rnds4" => "__builtin_ia32_sha1rnds4",
-            "llvm.x86.sha256msg1" => "__builtin_ia32_sha256msg1",
-            "llvm.x86.sha256msg2" => "__builtin_ia32_sha256msg2",
-            "llvm.x86.sha256rnds2" => "__builtin_ia32_sha256rnds2",
-
             "llvm.sqrt.v2f64" => "__builtin_ia32_sqrtpd",
             _ => unimplemented!("***** unsupported LLVM intrinsic {}", name),
```

Comment:
What do you think of putting the include here and moving the `unimplemented!()` in the generated file?